vuln/core/35.json

```diff
@@ -8,5 +8,5 @@
   "affectedEnvironments": [
     "all"
   ],
-  "severity": "unknown"
+  "severity": "critical"
```

Comment:
This also seems "high" https://nodejs.org/en/blog/vulnerability/cve-2015-8027_cve-2015-6764